encoder/vaapiencoder_h264.cpp

```diff
@@ -538,8 +538,12 @@ class VaapiEncStreamHeaderH264
             m_headers.insert(m_headers.end(), s, e);
             if (e == h.end())
                 break;
-            m_headers.insert(m_headers.end(), emulation, emulation + N_ELEMENTS(emulation));
+            
             s = e + N_ELEMENTS(zeros);
+            if (*s <= 3)
```

Comment:
Agree to add comments here, patch updated. But commit log is important for every developer, thus means "git blame"